incubator/hnc/pkg/controllers/hierarchy_controller_test.go

```diff
@@ -27,23 +27,23 @@ var _ = Describe("Hierarchy", func() {
 	})
 
 	It("should set a child on the parent", func() {
-		fooHier := newHierarchy(fooName)
-		fooHier.Spec.Parent = barName
-		updateHierarchy(ctx, fooHier)
+		fooHierarchy := newHierarchy(fooName)
```

Comment:
Please don't change the names of the variables for a few reasons:

1. It makes it much harder than it needs to be to review the relevant parts of this change.
2. It will make it more difficult to track changes that happened before and after this change.
3. Golang code style is to use short local variable names.

Reply:
Here we go. let me know if I missed anything.